.buildkite/pipeline.yml

```diff
@@ -7,6 +7,8 @@ x-common-params:
         # Allow WP-CLI to be run as root, otherwise it throws an exception.
         # Reference: https://git.io/J9q2S
         - 'WP_CLI_ALLOW_ROOT=true'
+        # Increase max available memory for node
+        - 'NODE_OPTIONS="--max-old-space-size=4096"'
```

Comment:
In the future, we could set this option directly in [the Docker image](https://github.com/wordpress-mobile/docker-gb-mobile-image/blob/trunk/Dockerfile).